java/com/epam/ta/salad/Repository/SQLService/SaladServiceImpl.java

```diff
@@ -1,4 +1,51 @@
 package com.epam.ta.salad.Repository.SQLService;
 
-public class SaladServiceImpl {
-}
+import Entities.Salad;
+import Entities.Vegetable;
+import com.epam.ta.parking.util.NullCheckUtil;
+import com.epam.ta.salad.Repository.Repository.SaladRepository;
+import com.epam.ta.salad.Repository.RepositorySpecification.VegetableSpecification;
+import com.epam.ta.salad.Repository.SQLRepository.VegetableRepositoryImpl;
+import com.epam.ta.salad.Repository.Service.SaladService;
+import exceptions.NullVegetableException;
+
+import java.sql.SQLException;
+import java.util.List;
+
+public class SaladServiceImpl implements SaladService {
+    @Override
+    public Salad add(Salad salad) throws SQLException, ClassNotFoundException {
+        return null;
+    }
+
+    @Override
+    public Salad update(Salad salad) throws SQLException {
+        return null;
+    }
+
+    @Override
+    public Salad remove(Salad salad) throws SQLException {
+        return null;
+    }
+
+    @Override
+    public List<Vegetable> getObjects(Salad salad) throws SQLException, NullVegetableException {
+        List<Vegetable> saladofVegetables = null;
+        if (null != salad) {
+            if (!NullCheckUtil.notNullCheck(String.valueOf((salad.getSalad())))){
+                throw new NullVegetableException("Salad is empty");
```

Comment:
it's a bad logic, we throw exceptions only in case if there is validation exception in CUD operations and our exceptions have the description of error. In your case, at first 'salad' haven't to be here. In any case with incorrect parameter you should return an empty list.